packages/build-info/src/frameworks/analog.test.ts

```diff
@@ -0,0 +1,30 @@
+import { beforeEach, expect, test } from 'vitest'
+
+import { mockFileSystem } from '../../tests/mock-file-system.js'
+import { NodeFS } from '../node/file-system.js'
+import { Project } from '../project.js'
+
+beforeEach((ctx) => {
+  ctx.fs = new NodeFS()
+})
+
+test('should detect analog', async ({ fs }) => {
+  const cwd = mockFileSystem({
+    'package.json': JSON.stringify({ dependencies: { '@analogjs/platform': '*' } }),
+  })
+  const detected = await new Project(fs, cwd).detectFrameworks()
+  expect(detected?.[0].id).toBe('analog')
+  expect(detected?.[0].build.command).toBe('ng build --prod')
+  expect(detected?.[0].build.directory).toBe('dist/analog/public')
+  expect(detected?.[0].dev?.command).toBe('ng serve')
+})
+
+test('should not detect angular', async ({ fs }) => {
+  const cwd = mockFileSystem({
+    'package.json': JSON.stringify({ dependencies: { '@analogjs/platform': '*', '@angular/cli': '*' } }),
+    'angular.json': '',
+  })
+  const detected = await new Project(fs, cwd).detectFrameworks()
+  const detectedFrameworks = (detected ?? []).map((framework) => framework.id)
+  expect(detectedFrameworks).not.toContain('angular')
+})
```

Comment:
this does feel weird to test here? angular test also feels weird - if there are better ideas for this kind of test which is dependent on both analog and angular detection - happy to adjust this

Reply:
Nope I think as this is a angular meta framework this is exactly the right test I was looking for!.

We want to detect the meta-framework not the framework in this case, and this test verifies that :)